test/cpp/api/parameterdict.cpp

```diff
@@ -0,0 +1,114 @@
+#include <gtest/gtest.h>
```

Comment:
Please modify the test if necessary after you finish modifying the parameterdict implementation.